x-pack/test/functional/es_archives/alerts/mappings.json

```diff
@@ -150,6 +150,9 @@
               },
               "type": "text"
             },
+            "notifyWhen": {
+              "type": "keyword"
+            },
```

Comment:
Why is this necessary here? Am I missing something?

Reply:
Any field that's inside a document in `data.json` needs to have a mapping defined. It just so happened that none of the other rule SO examples in `data.json` had `notifyWhen` but the SO I just added does, so I added the mapping. It's not strictly needed for this PR, I could remove it from the new doc I added to `data.json`

Reply:
ahhh gotcha.
Thanks Ying.... enjoy the weekend! ;)